flexmeasures/data/schemas/scheduling/__init__.py

```diff
@@ -317,6 +318,96 @@ def _to_currency_per_mwh(price_unit: str) -> str:
         return currency
 
 
+EXAMPLE_UNIT_TYPES: Dict[str, list[str]] = {
+    "energy-price": ["EUR/MWh", "JPY/kWh", "USD/MWh", "and other currencies."],
+    "power-price": ["EUR/kW", "JPY/kW", "USD/kW", "and other currencies."],
+    "power": ["kW"],
+}
+
+UI_FLEX_CONTEXT_SCHEMA: Dict[str, Dict[str, Any]] = {
+    "consumption-price": {
+        "default": None,  # Refers to default value of the field
+        "description": "Set the sensor that represents the consumption price of the site. This value will be used in the optimization.",
+        "allowed-units": EXAMPLE_UNIT_TYPES["energy-price"],
+    },
+    "production-price": {
+        "default": None,
+        "description": "Set the sensor that represents the production price of the site. This value will be used in the optimization.",
+        "allowed-units": EXAMPLE_UNIT_TYPES["energy-price"],
+    },
+    "site-power-capacity": {
+        "default": None,
+        "description": "This value represents the maximum power that the site can consume or produce. This value will be used in the optimization.",
+        "allowed-units": EXAMPLE_UNIT_TYPES["power"] + ["kVA", "MVA"],
+    },
+    "site-production-capacity": {
+        "default": None,
+        "description": "This value represents the maximum power that the site can produce. This value will be used in the optimization.",
+        "allowed-units": EXAMPLE_UNIT_TYPES["power"],
+    },
+    "site-consumption-capacity": {
+        "default": None,
+        "description": "This value represents the maximum power that the site can consume. This value will be used in the optimization.",
+        "allowed-units": EXAMPLE_UNIT_TYPES["power"],
+    },
+    "soc-minima-breach-price": {
+        "default": None,
+        "description": "This value represents the price that will be paid if the state of charge of a battery goes below the minimum state of charge. This must use the same currency and cannot be a negative price.",
```

Comment:
```suggestion
        "description": "This price helps to steer the scheduler solutions which do not breach state-of-charge minima. It has t be chosen in relation to actual energy costs encountered by your site ― too high, and it will dominate other constraints, too low and it will have no effect. Note that this setting will not actually result in actual costs we track (the real-life costs of breaching SoC minima are out of scope). Also, this setting must use the same currency as the other settings and cannot be a negative price.",
```

@Flix6x I believe this description is a much better way to describe these breach price settings. Please approve or adjust. Then @joshuaunity can apply to the other three settings.

Reply:
I think this is weaker as it still presents it as a price (while it is more of an internal nudge to steer the scheduler, not a real "price"). I foresee many users being confused.

Do you think that it can/will be used to model real-world prices of breaches?

Reply:
```suggestion
        "description": "This penalty value is used to discourage the violation of state-of-charge minima constraints, which the scheduler will attempt to minimize. It must use the same currency as the other settings and cannot be negative. While it's an internal nudge to steer the scheduler—and doesn't represent a real-life cost—it should still be chosen in proportion to the actual energy prices at your site. If it's too high, it will overly dominate other constraints; if it's too low, it will have no effect. Without this value, any infeasible state-of-charge minima would prevent a complete schedule from being computed. "```

@nhoening @Flix6x, what do you think?

Reply:
@nhoening @Flix6x, what do you think?